charts/seed-machines/charts/packet-machineclass/values.yaml

```diff
@@ -0,0 +1,16 @@
+machineClasses:
```

Comment:
Delete this file (the whole `packet-machineclass` (respectively, the whole `seed-machines) directory).